_posts/2020-03-17-updates.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: COVID-19 Days
+date: 2020-03-17  08:00:00
+categories: updates
+---
+
+![starburst Image](https://mcusercontent.com/845c4ebabb5b5ae7a6372c715/images/e1f0e196-2298-434d-ba15-8f94a0fa12c8.png)
+
+With the current global health crisis, our thoughts go out to all our colleagues and friends around the world: we hope that you are staying well, and safe.
+
+Here at our base is San Francisco, the Ilios project is continuing our work and development roadmap. At the same time, as a part of the UCSF community, we are doing our part to support the needs of our community, educational programs, and health services.
+
+While none of us can foresee quite what the next few months have in store for us all, at the very least we are hoping that we can provide some great positive notes amid the home isolations, EM and ICU reports, and the challenges of finding a roll of toilet paper at the local store.
+
+The features will continue to roll out, the fixes will continue to improve the experience, and Ilios will continue to provide a world-class framework for managing your curricula.
+
+Please [contact us](mailto:support@iliosproject.org), with any questions or interests.
+
+__STATUS__:
+- current version: __3.73.0__
+- next scheduled release: __03/20/2020__
```

Comment:
I'd guess this is going to bump a week based on what we currently have and what we need to get in to make a release worth it.

Reply:
bumped it a week.